gulpfile.js

```diff
@@ -26,7 +26,7 @@ const rename = require("gulp-rename");
 const replace = require("gulp-replace");
 const mkdirp = require("mkdirp");
 const path = require("path");
-const rimraf = require("rimraf");
+const { rimraf } = require("rimraf");
```

Comment:
`rimraf` is also used in two test files, see https://github.com/mozilla/pdf.js/search?q=rimraf, but those use the sychronous version. I assume those don't need to be updated then?

Reply:
Whoops, I somehow managed to overlook those when searching; thank you!

I've updated the imports for those cases as well, for consistency, and also removed the `disableGlob` option since that functionality was simply removed in version 4.